### ISSUE TITLE
Add contracts from query string via Etherscan

### ISSUE THREAD

**Issue by adrianmcli** [adrianmcli/eth95#4]
User pastes this into their browser:

```
https://eth95.dev/?address=0x6B175474E89094C44Da98b954EedeAC495271d0F
```

Immediately after page loads:

1. Grab ABI from Etherscan API (https://etherscan.io/apis#contracts)
2. Add contract object into local state
3. Display contract with its functions

**Comment by adrianmcli** [comment]
Thanks @kendricktan for the idea.

**Comment by adrianmcli** [comment]
Closed by #6
